lib/cli.mjs

```diff
@@ -556,11 +558,15 @@ export default async function main(argv) {
 
 	return standalone(options)
 		.then((linted) => {
-			if (!linted.output) {
+			if (!linted.output && !linted.code) {
```

Comment:
[suggest] How about refactoring the local variables like this to avoid repetition?

```suggestion
			const { output, code } = linted;

			if (!output && !code) {
```